server/src/test/java/io/crate/integrationtests/GeoShapeIntegrationTest.java

```diff
@@ -58,35 +63,41 @@ public class GeoShapeIntegrationTest extends IntegTestCase {
 
     @Before
     public void prepare() throws Exception {
-        execute("CREATE TABLE shaped (" +
+        for (var table : TABLE_TO_INDEX_TYPE_MAP.entrySet()) {
+            var tableName = table.getKey();
```

Comment:
Could you please  create all 4 columns (default geohash, geohash array, BKD, BKD array) in a single table?
Assertions are same anyway but we would also test how we can mix different index types in a single table.

Reply:
Sure. Done.